tokens/wstr.json

```diff
@@ -0,0 +1,10 @@
+{
+    "ethereum_address": "0xf0dc76c22139ab22618ddfb498be1283254612b1",
+    "aurora_address": "0xf34d508bac379825255cc80f66cbc89dfeff92e4",
+    "name": "WrappedStar",
+    "symbol": "WSTR",
+    "decimals": 18,
+    "icon": "data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiBzdGFuZGFsb25lPSJubyI/Pgo8IURPQ1RZUEUgc3ZnIFBVQkxJQyAiLS8vVzNDLy9EVEQgU1ZHIDIwMDEwOTA0Ly9FTiIKICJodHRwOi8vd3d3LnczLm9yZy9UUi8yMDAxL1JFQy1TVkctMjAwMTA5MDQvRFREL3N2ZzEwLmR0ZCI+CjxzdmcgdmVyc2lvbj0iMS4wIiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciCiB3aWR0aD0iMTAwMC4wMDAwMDBwdCIgaGVpZ2h0PSIxMDAwLjAwMDAwMHB0IiB2aWV3Qm94PSIwIDAgMTAwMC4wMDAwMDAgMTAwMC4wMDAwMDAiCiBwcmVzZXJ2ZUFzcGVjdFJhdGlvPSJ4TWlkWU1pZCBtZWV0Ij4KCjxnIHRyYW5zZm9ybT0idHJhbnNsYXRlKDAuMDAwMDAwLDEwMDAuMDAwMDAwKSBzY2FsZSgwLjEwMDAwMCwtMC4xMDAwMDApIgpmaWxsPSIjMDAwMDAwIiBzdHJva2U9Im5vbmUiPgo8cGF0aCBkPSJNMCA1MDAwIGwwIC01MDAwIDUwMDAgMCA1MDAwIDAgMCA1MDAwIDAgNTAwMCAtMzA3NSAwIGMtMjY4MCAwCi0zMDc2IC0yIC0zMDgxIC0xNSAtNSAtMTIgLTU5IC0xNCAtMzQ5IC0xNSAtMjY5IDAgLTM0NCAyIC0zNDggMTIgLTQgMTAgLTYKMTAgLTYgMSAtMSAtNyAtNiAtMTMgLTEyIC0xMyAtNSAwIC04IDQgLTUgOSAzIDUgLTEgMTEgLTkgMTQgLTkgNCAtMTIgMSAtOAotOSAzIC03IDAgLTE0IC02IC0xNCAtNiAwIC0xMSA3IC0xMSAxNSAwIDEzIC0xNzggMTUgLTE1NDUgMTUgbC0xNTQ1IDAgMAotNTAwMHogbTUwODAgMjM1OCBjMTggLTEzIDQxIC0zOSA1MSAtNTggMTggLTMzIDE5IC03OSAxOSAtOTg1IGwwIC05NTAgNjczCjY3MSBjMzY5IDM3MCA2ODIgNjc3IDY5NCA2ODMgMjYgMTMgODMgMTQgMTE1IDIgMzYgLTE0IDc3IC02MCA5MCAtMTAyIDI2IC04OAo0OSAtNjEgLTY4NSAtNzk3IGwtNjcyIC02NzQgOTU1IDQgYzcwNSAyIDk2MCAwIDk3NSAtOSA5MSAtNTIgMTE1IC0xNjAgNTMKLTIzOCAtNDUgLTU3IC03IC01NSAtMTAzNSAtNTUgbC05NDggMCA2NzkgLTY3OSA2NzkgLTY3OSA1IC01NiBjNCAtNDggMSAtNjIKLTIxIC05NSAtNDUgLTY3IC0xMjQgLTg5IC0xOTUgLTUyIC0yMCAxMSAtMzMyIDMxNSAtNjk3IDY4MCBsLTY2MCA2NjEgLTUKLTk2NCAtNSAtOTY0IC0yNSAtMjcgYy03OCAtODMgLTIwMCAtNzEgLTI1MSAyNSAtMTggMzMgLTE5IDc5IC0xOSA5ODUgbDAgOTUwCi02NzIgLTY3MSBjLTM3MCAtMzcwIC02ODMgLTY3NyAtNjk1IC02ODMgLTMwIC0xNSAtODkgLTE0IC0xMjIgNCAtNDAgMjAgLTc5CjcwIC04NyAxMTAgLTE0IDc4IC0yMyA2OCA2OTAgNzgzIGw2NzEgNjczIC05NTAgLTIgYy02MTUgLTEgLTk1OSAyIC05NzUgOQotODIgMzMgLTExNyAxNDkgLTY3IDIyMyA0OSA3MyAtMTggNjkgMTA0MiA2OSBsOTUwIDAgLTY3MSA2NzMgYy0zNzAgMzY5IC02NzcKNjgyIC02ODMgNjk0IC0xNiAzMSAtMTQgOTAgNSAxMjkgMTggMzcgNjQgNzEgMTEyIDgwIDc0IDE1IDY5IDIwIDc4MCAtNjkwCmw2NzIgLTY3MSAwIDk1MSBjLTEgNTY4IDMgOTYwIDkgOTc1IDkgMjUgNDIgNTkgNzYgNzcgMzUgMjAgMTA5IDE0IDE0NSAtMTB6Ii8+CjxwYXRoIGQ9Ik0zNzkwIDk5OTYgYzAgLTIgOSAtNiAyMCAtOSAxMSAtMyAyMCAtMSAyMCA0IDAgNSAtOSA5IC0yMCA5IC0xMSAwCi0yMCAtMiAtMjAgLTR6Ii8+CjwvZz4KPC9zdmc+Cg==",
```

Comment:
It's better to use an optimized URL. Please use something like this [tool](https://yoksel.github.io/url-encoder/)

Reply:
Changed URL